ecommerce/mail_api.py

```diff
@@ -93,3 +97,25 @@ def send_course_run_enrollment_email(enrollment):
         )
     except:  # pylint: disable=bare-except
         log.exception("Error sending enrollment success email")
+
+
+def send_course_run_unenrollment_email(enrollment):
+    """
+    Notify the user of successful unenrollment for a course run
+
+    Args:
+        enrollment (CourseRunEnrollment): the enrollment for which to send the email
+    """
+    try:
+        user = enrollment.user
+        api.send_message(
+            api.message_for_recipient(
+                user.email,
+                api.context_for_user(
+                    user=user, extra_context={"enrollment": enrollment}
+                ),
+                EMAIL_COURSE_RUN_UNENROLLMENT,
+            )
+        )
+    except:  # pylint: disable=bare-except
+        log.exception("Error sending unenrollment success email")
```

Comment:
Wouldn't it be better to set extra information in this log? For example, the enrollment used for sending out the email?